### PR TITLE
Chart.4

### PR DESCRIPTION
**Pull request by olu144**
I have added a line chart that displays amazon stock data from a csv

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/AmazonServlet.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns bigfoot data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
+@WebServlet("/amazon-data")
+public class AmazonServlet extends HttpServlet {
+  private LinkedHashMap<String, Float> AmazonData = new LinkedHashMap<>();
```
Comment (dborkan) [inline portfolio/src/main/java/com/google/sps/servlets/AmazonServlet.java]: nit, this should start with lowercase, like `amazonData`.  I believe starting with uppercase is reserved for classnames (like LinkedHashMap)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -79,28 +79,28 @@ async function isLoggedIn() {
...
-  ]);
-  const options = {
-    'title': 'Zoo Animals',
-    'width':500,
-    'height':400
-  };
-  const chart = new google.visualization.PieChart(document.getElementById('chart-container'));
-  chart.draw(data, options);
+  fetch('/amazon-data').then(response => response.json())
+  .then((AmazonData) => {
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: nit: this should also be `amazonData`

**Review by dborkan — APPROVED** [review]
Good work figuring out how to read CSVs!! Just a few small nits, otherwise good to merge

File: portfolio/src/main/java/com/google/sps/servlets/AmazonServlet.java (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/java/com/google/sps/servlets/AmazonServlet.java]: addressed in "fixed case of a in amazonData"

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (olu144) [reply portfolio/src/main/webapp/script.js]: addressed in "fixed case of a in amazonData"

**Comment by olu144** [comment]
fixes from chart.3 have been merged

**Review by dslucas — APPROVED** [review]
Cool stuff!

**Pull request merged**